metricbeat/module/elasticsearch/node_stats/_meta/docs.asciidoc

```diff
@@ -1,5 +1,3 @@
-=== Elasticsearch node_stats metricset
```

Comment:
This is cleanup and should not have been here. First though was this cause the doc build issue.